### ISSUE TITLE
[Plug-in] window.withProgress shows up multiple times in window footer

### ISSUE THREAD

**Issue by invincibleJai** [theia-ide/theia#4499]
### Description
I was trying to test https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics in theia and I observed status messages are shown multiple time, instead of updating messages in same window loaction as happens in VSCode

### Reproduction Steps
- Install https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics with `plugin by ID`
- open `package.json`
- start editing  `package.json`
- Observe status message in the bottom window bar

**OS and Theia version:**
MAC, and using image `heiaide/theia-full:next`
**Diagnostics:** 
Progress message is being called multiple times with 

```
vscode.window.withProgress(
            {
              location: vscode.ProgressLocation.Window,
              title: StatusMessages.EXT_TITLE
            }
...
```

Check window footer in screenshots

<img width="1673" alt="screenshot 2019-03-07 at 3 09 44 pm" src="https://user-images.githubusercontent.com/5129024/53947084-15475780-40eb-11e9-8073-fd12964e0aad.png">

**Comment by akosyakov** [comment]
Has not it been already fixed: https://github.com/theia-ide/theia/pull/4480

I cannot reproduce it on latest master.

**Comment by invincibleJai** [comment]
@akosyakov thanks for updating ,  am using image `theiaide/theia-full:next`

Can you pls help with some doc to build from src, I tried some time back but couldn't

**Comment by akosyakov** [comment]
@invincibleJai It is build nightly, should be there, maybe some caching

**Comment by invincibleJai** [comment]
thanks , let me try again

**Comment by akosyakov** [comment]
> Can you pls help with some doc to build from src, I tried some time back but couldn't

Have you seen https://github.com/theia-ide/theia/blob/master/doc/Developing.md#quick-start?

**Comment by invincibleJai** [comment]
@akosyakov thanks, it's working am not seeing duplication now. Closing it.
